lang/cs/BUILD.md

```diff
@@ -24,7 +24,13 @@ Prerequisites
 -------------
 
   * Windows OS.
-  * [Building REEF Java](../java/BUILD.md).
+  * Apache maven
```

Comment:
These exact setup steps need to be performed to do a standalone Java build. If getting this information from Java build instructions is non-obvious, we need to change it to be obvious there (and to make it obvious that Java build is a requirement for .NET build, not a separate process), not to duplicate Java build instructions in .NET build instructions.